src/main/java/com/quaz/ticket/screening/ScreeningSeatEmbeddedResponse.java

```diff
@@ -0,0 +1,15 @@
+package com.quaz.ticket.screening;
+
+import java.io.Serializable;
+
+public record ScreeningSeatEmbeddedResponse(
```

Comment:
Mam wrażenie, że trochę jest bałagan z tymi pakietami. Ta klasa ma swój dedykowany pakiet `screeningseat`. Za to klasa `Seat` jest w pakiecie `screeningroom`. Nie jestem fanem takiego podziału, o ile jest prosto przypisywać kolejne klasy do pakietów, to jednak później ciężko się tam odnaleźć z punktu widzenia biznesowego/domenowego. Jak Ty uważasz?

Reply:
Ogólnie staram się odchodzić od podejścia trójwarstwowego w moich projektach na rzecz podziału bardziej domenowego/biznesowego. Jeżeli chodzi o klasę `ScreeningSeatEmbeddedResponse` miałem pewien problem, normalnie staram się unikać zagnieżdżonych kolekcji w obiektach, które zwracam na rzecz oddzielnego endpointu (w tym wypadku byłoby wylistowanie wszystkich screening seatów dla danego screeningu). Aczkolwiek w tym wypadku chciałem to zawrzeć w jednym ze względu na to, że miało to być prezentowane w konsoli. Wyżej wymieniona klasa jest wykorzystywana tylko i wyłącznie w ramach obiektu `ScreeningDetails`, dlatego ostatecznie uznałem, że jej miejsce powinno być w tym pakiecie, pomimo tego, że oryginalnie było w `screeningseat`. To samo się tyczy Seat w pakiecie `screeningroom`, klasa `Seat` nie posiada żadnego mappera ani repozytorium, i jest dostępna tylko z klasy `ScreeningRoom` co było celowym zabiegiem, w momencie, w którym zrobiłbym kontroler, serwis lub repozytorium to przeniósłbym to do oddzielnego pakietu. Czy to jest prawidłowe podejście to ciężko mi powiedzieć, na ten moment badam jak takie podziały pakietowe sprawdzają się w użyciu. Dzięki za uwagę, jeżeli taki podział sprawia wrażenie bałaganu to oznacza, że można by coś posegregować aby było bardziej intuicyjnie.